Code/CheXpertDataset.py

```diff
@@ -29,18 +30,13 @@
     'Support Devices'
 ]
 
-
 class ChexpertDataset(Dataset):
     def __init__(self,
                  csv_file,
                  root_dir,
                  image_transform=None,
                  label_transform=None):
-        df = pd.read_csv(csv_file)
-
-        # Only use frontal images
-        self.df = df[df["Frontal/Lateral"] == "Frontal"]
-
+        self.df = pd.read_csv(csv_file)

```

Comment:
Decided based on the Chexpert inputs we probably don't need to filter out the Frontal/Lateral view distinction after all. 

Believe this is because of the FixedRatioResize mechanism below.

Reply:
Do you know why this is?

Reply:
As it'll handle resizing the images and keeping aspect ratio the model it scales it correctly. I didn't see anything in they're codebase that was removing the Lateral images. It looked like they simply trained on both. I do encourage you to take a second look though. I could certainly be wrong on that one.

Reply:
you also may find this interesting... i hadn't though of the problem this way... they applied a different transform per label. 
https://github.com/jfhealthcare/Chexpert/blob/47e1bf7f8ed2b95c8985ae43801ffffde92fa205/data/dataset.py#L11

Took me awhile to figure out what it did. not a lot of comments in their code.